qemu.mk

```diff
@@ -32,6 +32,19 @@ all-clean: bios-qemu-clean busybox-clean linux-clean optee-os-clean \
 ################################################################################
 # QEMU
 ################################################################################
+QEMU_CONFIGURE_PARAMS = --target-list=arm-softmmu --cc="$(CCACHE)gcc" \
+						--extra-cflags="-Wno-error"
+
+QEMU_VIRTFS_ENABLE		?= n
```

Comment:
what about just passing mount point to alias as an additional param:
```
alias mount_shared='mount -t 9p -o trans=virtio host'
```
Usage:
`$ mount_shared /mnt/shared`